data/sql/updates/pending_db_world/rev_1680443009841196300.sql

```diff
@@ -0,0 +1,6 @@
+-- Ferra loot update
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(14308, 4500, 0, 1, 0, 1, 0, 1, 1, 'Ferra - Traveler\'s Backpack');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(14308, 24016, 24016, 1, 0, 1, 0, 1, 1, 'Ferra - (ReferenceTable)');
```

Comment:
You always need a DELETE QUERY before an INSERT, see: https://www.azerothcore.org/wiki/sql-standards#insert-amp-delete